pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>io.gravitee.policy</groupId>
     <artifactId>gravitee-policy-retry</artifactId>
-    <version>2.0.0</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
It must stay `2.0.0`, the version will be bumped automatically during the release